Configuration/PyReleaseValidation/python/MatrixInjector.py

```diff
@@ -65,6 +65,25 @@ def __init__(self,opt,mode='init',options=''):
         if(opt.batchName):
             self.batchName = '__'+opt.batchName+'-'+self.batchTime
 
+        ####################################
+        # Checking and setting up GPU attributes
+        ####################################
+        # Mendatory
+        self.RequiresGPU = opt.RequiresGPU
+        if self.RequiresGPU not in ('forbidden','optional','required'):
+            print('RequiresGPU must be forbidden, optional, required. Now, set to forbidden.')
+            self.RequiresGPU = 'forbidden'
+        if self.RequiresGPU == 'optional':
+            print('Optional GPU is turned off for RelVals. Now, changing it to forbidden')
+            self.RequiresGPU = 'forbidden'
```

Comment:
```suggestion
```
otherwise, doesn't this completely defeat the purpose of "optional" ?

Reply:
If we would like to keep optional as real optional definition also in relvals, this can be removed. No issue on removal. I will include it in the next commit.

Reply:
Well, if we don't want to use "optional" in the RelVals, we should simply not use it as an option when creating and submitting them.

Additionally, what I do not understand is how "optional" could be used for any other uses case if it is always removed by `runTheMatrix` ?